src/OrderFormAcceptanceTests.Objects/GlobalSuppressions.cs

```diff
@@ -0,0 +1,8 @@
+﻿// This file is used by Code Analysis to maintain SuppressMessage
+// attributes that are applied to this project.
+// Project-level suppressions either have no target or are given
+// a specific target and scoped to a namespace, type, member, etc.
+
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("Performance", "CA1822:Mark members as static", Justification = "By selectors need to be non-static members", Scope = "namespaceanddescendants", Target = "~N:OrderFormAcceptanceTests.Objects.Pages")]
```

Comment:
Why do `By` selectors need to be instance methods? Having a look at `Pages.AdditionalServices` could it be that you just need to disambiguate between the `Pages` field and the namespace. For example, would using `Object.Pages.AdditionalServices.AddAdditionalServices` work in `AddAdditionalServiceButtonDisplayed`?